CODE_OF_CONDUCT.md

```diff
@@ -15,7 +15,7 @@ diverse, inclusive, and healthy community.
 ## Our Standards
 
 Examples of behavior that contributes to a positive environment for our
-community include:
+community includes:
```

Comment:
Thanks for submitting this PR. This should be `include`. I'm happy to merge your other suggestions below if you revert this part.